_episodes/03-types-conversion.md

```diff
@@ -81,7 +81,7 @@ TypeError: unsupported operand type(s) for -: 'str' and 'str'
 ~~~
 {: .error}
 
-## Strings can be added and multiplied.
+## You can use the "+" and "\*" operators on strings.
```

Comment:
The backslash was there to escape the `*` so Markdown won't interpret it as beginning an italic section. It won't be rendered in the output. But it doesn't seem to be necessary, so I'm all for its removal :+1: